lib/routes.js

```diff
@@ -134,6 +135,7 @@ function postTx (req, res, next) {
       return res.json(tx)
     })
     .catch(err => {
+      if (err.code === SERIALIZATION_FAILURE_CODE) return res.status(204).json({})
```

Comment:
Can you add a comment explaining why it's OK to return a 204 on this error? You can put this in a new PR, since I'm merging this.